slam/src/main.rs

```diff
@@ -132,7 +134,7 @@ fn main() {
             .filter_map(|conn| conn.fetch_block_info(empty()).ok())
             .map(|block_info| block_info.minimum_fee)
```

Comment:
I'm wondering, since the old networks dont provide a minimum_fee, wouldn't this be zero (because protobuf), but the `.max()` call will still succeed?

Reply:
Yep, you're right, great catch.